src/moonstone-samples/lib/LightPanelsSample.js

```diff
@@ -38,7 +61,14 @@ module.exports = kind({
 				}
 			],
 			clientComponents: [
-				{kind: Scroller, style: 'height:800px', components: [
+				{kind: Control, classes: 'control-container', components: [
+					{kind: Control, components: [
+						{kind: IconButton, name: 'previous', icon: 'arrowlargedown', ontap: 'prevIconTapped'},
```

Comment:
Good catch! I fixed one of these earlier and (incorrectly) assumed I had taken care of them all.